packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -821,6 +836,49 @@ class Auth {
         return token
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizePasswordless.
+     */
+    async authorizePasswordless(parameters: AuthorizePasswordlessParams) {
+        const userid = parameters.userid
+        const mode = this.passwordlessConfig.mode === 'sms' ? 'sms' : 'callback'
```

Comment:
You might want to use CONSTANTS for 'sms' and 'callback'. Better yet, these values should be exported from the isomophic sdk.

Reply:
Is there a way to export these values from the isomorphic sdk? I only see the values defined [here](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/main/apis/shopper-login/shopper-login.raml#L63-L66) in the raml.